examples/js/loaders/RGBELoader.js

```diff
@@ -391,3 +391,37 @@ THREE.RGBELoader.prototype.setType = function ( value ) {
 	return this;
 
 };
+
+THREE.RGBELoader.prototype.load = function ( url, onLoad, onProgress, onError ) {
+
+	function onLoadCallback( texture, texData ) {
+
+		switch ( texture.type ) {
+
+			case THREE.UnsignedByteType:
+
+				texture.encoding = THREE.RGBEEncoding;
+				texture.minFilter = THREE.NearestFilter;
+				texture.magFilter = THREE.NearestFilter;
+				texture.generateMipmaps = false;
+				texture.flipY = true;
+				break;
+
+			case THREE.FloatType:
```

Comment:
Can it be `THREE.HalfFloatType` too?

Reply:
`HalfFloatType` support will be in the next PR !

Reply:
Sweet!